examples/measure_download_time/measure_download_time.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python3
+import asyncio
+import base64
+import pathlib
+import sys
+from datetime import datetime, timedelta
+
+from yapapi import Golem
+from yapapi.services import Service
+from yapapi.log import enable_default_logger
+from yapapi.payload import vm
+
+examples_dir = pathlib.Path(__file__).resolve().parent.parent
+sys.path.append(str(examples_dir))
+
+from utils import (build_parser, run_golem_example, TEXT_COLOR_CYAN, TEXT_COLOR_DEFAULT, TEXT_COLOR_RED)
+
+task_finished_event = asyncio.Event()
+
+class ApiCallService(Service):
+    def __init__(self, *args, url: str, **kwargs):
+        super().__init__(*args, **kwargs)
+        self._url = url
+
+    @staticmethod
+    async def get_payload():
+        manifest = open("manifest.json", "rb").read()
+        manifest = base64.b64encode(manifest).decode("utf-8")
+
+        manifest_sig = open("manifest.json.base64.sign.sha256", "rb").read()
+        manifest_sig = base64.b64encode(manifest_sig).decode("utf-8")
+
+        manifest_sig_algorithm = "sha256"
+
+        # both DER and PEM formats are supported
+        manifest_cert = open("foo_req.cert.pem", "rb").read()
+        manifest_cert = base64.b64encode(manifest_cert).decode("utf-8")
+
+        return await vm.manifest(
+            manifest=manifest,
+            manifest_sig=manifest_sig,
+            manifest_sig_algorithm=manifest_sig_algorithm,
```

Comment:
It is a similar problem regarding having an enum vs passing a string literal.
This "sha256" goes directly to `openssl` lib. It ended up this way, because the lib did not provide list of supported algorithms. Probably it should be changed to an enum.

Reply:
hmm, will think about it...

Reply:
hmm... after a thought, I'd say _here_ the library should be completely agnostic of the signing algorithm(s) in use and should just accept a clear string